website/versioned_docs/version-5.x/module-types.md

```diff
@@ -1,51 +1,53 @@
-When you call `mountParcel` or `mountRootParcel` [(see api)](./parcels-api.md) the parcel is mounted immediately and returns the parcel object. You need to call the `unmount` method on the parcel manually when the component that calls `mountParcel` unmounts.
+### You manage the lifecycles of parcels
+When you call `mountParcel` or `mountRootParcel` [(see API)](./parcels-api.md) the parcel is mounted immediately and returns the parcel object. You need to call the `unmount` method on the parcel manually when the component that calls `mountParcel` unmounts.
 
-### Parcels are best suited for sharing UI between frameworks
+### Parcels are best suited for sharing pieces of UI between frameworks
 Creating a parcel is as easy as using the [single-spa helpers](./ecosystem#help-for-frameworks.md) for that framework on a specific component/UI. This returns an object (`parcelConfig`) that single-spa can use to create and mount a parcel.
 Because single-spa can mount a parcel anywhere, this gives you a way to share UI/components across frameworks. It should not be used if the shared UI is being used in another application of the same framework.
-For example: `applicationOne` is written in Vue and contains all the UI/Logic to create a user. `application2` is written in React and needs to create a user. Using a single-spa parcel allows you to wrap your `app1` Vue component
-in a way that will make it work inside `application2` despite the different frameworks. Even better if `application2` is unmounted by single-spa (per the activity function returning false)
+For example: `application1` is written in Vue and contains all the UI and logic to create a user. `application2` is written in React and needs to create a user. Using a single-spa parcel allows you to wrap your `application1` Vue component
+in a way that will make it work inside `application2` despite the different frameworks.
 Think of parcels as a single-spa specific implementation of webcomponents.
 
-## Utility modules share common logic
+## Utilities
+
+### Utility modules share common logic
 Utility modules are a great place to share common logic. Instead of each application creating their own implementation of common logic, you can use a plain JavaScript object (single-spa utility) to share that logic.
 For example: Authorization. How does each application know which user is logged in? You could have each application ask the server or read a JWT but that creates duplicate work in each application.
-Using Utility modules you can implement logic around who is logged in one module (with all the necessary methods exported on the module) and each single-spa application can use the logic by importing the methods from the utilty module.
+Using utility modules, you can implement logic around who is logged in all in one module (with all the necessary methods exported on the module) and each single-spa application can use the logic by importing the methods from the utilty module.
```

Comment:
this line still seems awkward. 

perhaps `Using the utility module pattern allow you to create one module that implements the logic around who is logged in`

or something. It's a tough sentence to get right 🤷‍♀️

Reply:
Sure, we can make it better. I just pushed up a new commit. How about this?

> Using the utility module pattern would allow you to create one module that implements the authorization logic. This module would export any needed methods, and then your other single-spa applications could use those authorization methods by importing them.